proto/umee/lend.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
Two notes on this file:

1. By Convention, all messages and txs related proto schemas should live in a single `tx.proto` file
2. Let's add some comment docs to each message type

e.g. https://github.com/cosmos/cosmos-sdk/blob/master/proto/cosmos/bank/v1beta1/tx.proto

Reply:
Ohh also, I think they need to be nested under a version too,

e.g. `proto/umee/v1beta1/lend.proto`

Reply:
Moved the `tx.go` (was lend.go), `genesis.go`, and `query.go` into `proto/umee/leverage/v1beta1`.

The leverage term seems to be needed to specify module (so if we add another module under `x/` which needs proto files, they would be separate.